skeleton/aws/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "terraform_organization" {
```

Comment:
How about adding the type also? we discussed here https://github.com/nimblehq/infrastructure-templates/pull/27/files#r878971645

Reply:
Thanks @junan I updated it in https://github.com/nimblehq/infrastructure-templates/pull/26/commits/13a256d0583cfb83d7fd60060a7223203e70a142